addons/fieldmanual/config.cpp

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+
+class CfgPatches
+{
+    class ADDON
+    {
```

Comment:
```suggestion
class CfgPatches {
    class ADDON {
```

Opening brace on same line.